<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_candidats</class>
 <widget class="QDialog" name="gestion_candidats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabwidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1031</width>
     <height>521</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Consulter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Ajout">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PRENOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>EMAIL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TELEPHONE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CIN">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>34</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>410</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>S'inscrire comme candidat</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>430</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion Enseignants</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>350</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE NAISSANCE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NOM">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TELEPHONE">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_EMAIL">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PRENOM">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Consulter">
    <attribute name="title">
     <string>Consulter</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_CIN_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>18</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PRENOM_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>150</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>29</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CLASSE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>260</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ADRESSE_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>200</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PRENOM</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_NOM_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>90</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MARCHANDISE_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>260</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>441</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Modifier">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Supprimer">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>330</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="triC">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>TRI par Classe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_triN">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>TRI pan Nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_triA">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>370</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>TRI par CIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recherche">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>430</y>
       <width>171</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche par Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rech">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>430</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="inprimer">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>390</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statistique">
     <property name="geometry">
      <rect>
       <x>531</x>
       <y>450</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>stat</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
